### PR TITLE
Splash screen while login the user and retrieving access tokens

### LINKED ISSUE #217: Add a splash screen while the app is loading/silently login in
Currently it seems like the app takes a few seconds to appear as it is silently login.

### PR DESCRIPTION
**Pull request by timotheeguerin**
fix #217

### CODE REVIEW

File: app/services/adal/user-authorization.ts [hunk 1]
```diff
@@ -1,10 +1,12 @@
-import { remote } from "electron";
+import * as electron from "electron";
```
Comment (paselem) [inline app/services/adal/user-authorization.ts]: Why did you change this? What additional objects are you using from "electron" in this class?

Comment (timotheeguerin) [inline app/services/adal/user-authorization.ts]: none actually it was just conflicting with the remote service name I inject
